R/add_to_view.R

```diff
@@ -70,14 +71,14 @@ add_to_view <- function(object, call.args, side, where){
 #' gsplot:::filter_arguments('points', x=1:5, y=1:5, xlim=c(0,10), ylim=c(0,10), 
 #'                callouts(labels=c(rep(NA, 4), "oh")))$extracted.args
 #' @keywords internal
-filter_arguments <- function(fun.name, ...){
+filter_arguments <- function(fun.name, ..., custom.config = FALSE){
```

Comment:
was thinking that default would be `custom.config = NULL` since that is what you would get if 
`custom.config = object[["global"]][["config"]][["config.file"]]` didn't exist. Maybe I missed it, but where do you test for `!custom.config` or where is that used?